core/src/main/scala/org/apache/pekko/persistence/r2dbc/ConnectionFactoryProvider.scala

```diff
@@ -18,13 +18,16 @@ import java.util.concurrent.ConcurrentHashMap
 
 import scala.concurrent.Future
 import scala.concurrent.duration.Duration
+import scala.util._
```

Comment:
is it possible to use fully specified imports instead of the wildcard?

Reply:
Fixed